specification/dimensions/tags.md

```diff
@@ -0,0 +1,76 @@
+# Tags
+
+The Tags column represents the set of finalized user-defined and/or provider-defined tags assigned to Tag Sources.  Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges.
+
+A tag becomes 'finalized' when a single value is selected from a set of possible tag values assigned to the tag key at different levels of a resource grouping hierarchy (if supported by the Provider).
+
+The Tags column adheres to the following requirements:
+
+* The Tags column MUST contain user-defined and/or provider-defined tags.
+* The Tags column MUST only contain finalized tags.
+* The Tags column MUST be in Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
```

Comment:
In my experience, I have never seen a tag without value, and I would not not know what to do with it.
I would be interested to have an example or link if possible.
In any case, feel free to resolve this comment.

Reply:
https://cloud.google.com/vpc/docs/add-remove-network-tags